projects/backend/fronts.ts

```diff
@@ -257,29 +257,38 @@ const getDisplayName = (front: string) => {
     return split.charAt(0).toUpperCase() + split.slice(1)
 }
 
-export const getFront = async (
-    issue: string,
-    id: string,
+const fetchPublishedIssue = async (
+    edition: string,
+    issueDate: string,
+    frontId: string,
     source: string,
     lastModifiedUpdater: LastModifiedUpdater,
-): Promise<Attempt<Front>> => {
+): Promise<Attempt<PublishedIssue>> => {
     const path: Path = {
-        key: `daily-edition/${issue}/${source}.json`,
+        key: `${edition}/${issueDate}/${source}.json`,
```

Comment:
while we are here, `source` could be renamed to `version`

Reply:
done